Gemfile

```diff
@@ -39,13 +39,15 @@ gem 'kaminari', '~> 1.1'
 
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
+  gem 'aws-sdk-s3'
```

Comment:
I don't think this is supposed to be in the development or test group because then we won't be able to use S3 in production